_inc/lib/core-api/class.jetpack-core-api-site-endpoints.php

```diff
@@ -69,6 +69,17 @@ public static function get_related_posts( $api_request ) {
 			);
 		}
 
+		// Activate the Related Posts module if it's not currently active
+		if ( ! Jetpack::is_module_active( 'related-posts' ) ) {
+			if ( ! Jetpack::activate_module( 'related-posts' ) ) {
```

Comment:
Nit: could these two `if`s be combined to one?

Reply:
Maybe your question means we should make this statement more readable?